data/translations/ja.ts

```diff
@@ -69,5 +69,49 @@
         <source>Select your user and enter password</source>
         <translation>ユーザ名を選び、パスワードを入力してください</translation>
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
+    </message>
+    <message>
+        <source>Change password</source>
+        <translation>パスワードを変更</translation>
+    </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>ユーザ名を入力</translation>
+    </message>
+    <message>
+        <source>Enter your password</source>
+        <translation>パスワードを入力</translation>
+    </message>
+    <message>
+        <source>Password:</source>
+        <translation>パスワード:</translation>
+    </message>
+    <message>
+        <source>Show password</source>
+        <translation>パスワードを表示する</translation>
+    </message>
+    <message>
+        <source>Hide password</source>
+        <translation>パスワードを隠す</translation>
+    </message>
+    <message>
+        <source>Suspend</source>
+        <translation>サスペンド</translation>
+    </message>
+    <message>
+        <source>Hibernate</source>
+        <translation>ハイバネート</translation>
+    </message>
+    <message>
+        <source>Password change aborted because maximum tries reached</source>
+        <translation>最大試行回数に達しましたため、パスワード変更は中止しました</translation>
+    </message>
+    <message>
+        <source>New password change round! Please input current password again!</source>
+        <translation>新しいパスワード変更ラウンドです! 今のパスワードをもう一度入力してください!</translation>
```

Comment:
@kisaragi-hiu
"パスワード変更ラウンド" sounds a bit unnatural, but I'm not sure how to make it natural because I don't know the context.
Do you know where this text is used?
(Screenshot is much appreciated if you have)

Reply:
Is this text used anywhere?
I suspect this text is no longer used.

Reply:
I'm not sure this is used anywhere either. Searches for the text or [the key](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml#L50) on GitHub just land back at [TextConstants.qml](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml) in this repository.

The name of the key suggests it might've been meant for [PAM lockout (max tries)](https://wiki.archlinux.org/title/Security#Lock_out_user_after_three_failed_login_attempts) at some point, but then the text just doesn't make sense.

Reply:
Or we have another option to remove the Japanese translation for now.

Reply:
FWICT it's used in the stale `expired-password` branch and was added such that themes can use some SDDM provided translations. I don't think it's used at all ATM.

Reply:
I've removed these two strings from the PR.